ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -183,6 +183,8 @@ def aws_cleanup():
 
     if not clusters_to_delete:
         logger.info("No clusters to delete")
+    else:
+        get_openshift_installer()
```

Comment:
@clacroix12 ,
Will there be any scenario where cluster is bit old and openshift-installer is not compatible ? Just a thought. Otherwise LGTM.

Reply:
Potentially, but I think that's more a question for the openshift devs. If the latest installer won't work for some versions of 4.x then we should know about that and try to do some analysis of the cluster before destroying. Currently though, we don't do any such thing and this change is just to ensure that we have the openshift installer ready for the destroy command we try to execute. If we do hit any issues like that then we can open a ticket and come up with a fix.